### PR TITLE
MC-40013: Payment Methods radio-buttons disappear after a "Create New Order" page in Admin panel is reload

### LINKED ISSUE #32106: Payment Methods radio-buttons disappear after a "Create New Order" page in Admin panel is reload
**Preconditions:**
* Magento 2.4.1, 2.4.2 is installed
* A Customer Account is created with Billing and Shipping address
* A Simple product is created
* Several payment methods are configured

**Steps to Reproduce:**
* Go to the Magento Admin Panel > SALES > Orders
* Click "Create New Order" button
* Select the created Customer
* Click "Add Products" button
* Select created Production
* Click "Add Selected Product(s) to Order" button
* Scroll the page to the "Payment & Shipping Information" section
* Reload the page

**(/) Expected results:**

Radio-buttons of Payment Methods should not disappear

**(x) Actual results:**

Radio-buttons of Payment Methods disappear

### PR DESCRIPTION
**Pull request by viktorpetryk**
### Fixed Issues (if relevant)

1. Fixes magento/magento2#32106

### Manual testing scenarios (*)

see magento/magento2#32106

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by victorpetryk** [comment]
@magento run all tests

**Comment by victorpetryk** [comment]
@magento run all tests

**Comment by victorpetryk** [comment]
@magento run all tests

File: app/code/Magento/Sales/view/adminhtml/templates/order/create/billing/method/form.phtml [hunk 1]
```diff
@@ -4,57 +4,68 @@
...
-            <?= $block->getChildHtml('payment.method.' . $_code) ?>
-        </dd>
-    <?php endforeach; ?>
-    </dl>
-</div>
+                    <label class="admin__field-label" for="p_method_<?= $escaper->escapeHtmlAttr($_code); ?>">
+                        <?= $escaper->escapeHtml($_method->getTitle()) ?>
+                    </label>
+
+                    <?php if ($_methodsCount > 1): ?>
```
Comment (zakdma) [inline app/code/Magento/Sales/view/adminhtml/templates/order/create/billing/method/form.phtml:55]: Should cover these changes with MFTF test

**Review by zakdma — CHANGES_REQUESTED** [review]

**Comment by victorpetryk** [comment]
@magento run all tests

File: app/code/Magento/Sales/Test/Mftf/Section/AdminOrderFormPaymentSection.xml [hunk 2]
```diff
@@ -29,5 +29,6 @@
         <element name="purchaseOrderOption" type="radio" selector="#p_method_purchaseorder" timeout="30"/>
         <element name="purchaseOrderNumber" type="input" selector="#po_number"/>
         <element name="freePaymentLabel" type="text" selector="#order-billing_method_form label[for='p_method_free']"/>
+        <element name="paymentLabelWithRadioButton" type="text" selector="#order-billing_method_form .admin__control-radio+label"/>
```
Comment (zakdma) [inline app/code/Magento/Sales/Test/Mftf/Section/AdminOrderFormPaymentSection.xml]: Can we use parametrized selector to check payment radio by name

File: app/code/Magento/Sales/Test/Mftf/ActionGroup/AssertAdminPaymentMethodRadioButtonExistsOnCreateOrderPageActionGroup.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminPaymentMethodRadioButtonExistsOnCreateOrderPageActionGroup">
+        <annotations>
+            <description>Checks the payment method radio button presents on the Admin Create Order page.</description>
+        </annotations>
+        <conditionalClick selector="{{AdminOrderFormPaymentSection.linkPaymentOptions}}" dependentSelector="{{AdminOrderFormPaymentSection.linkPaymentOptions}}" visible="true" stepKey="clickGetAvailablePaymentMethods"/>
+        <waitForElementVisible selector="{{AdminOrderFormPaymentSection.paymentBlock}}" stepKey="waitForPaymentOptions"/>
+        <seeElement selector="{{AdminOrderFormPaymentSection.paymentLabelWithRadioButton}}" stepKey="seeLabelWithRadioButton"/>
```
Comment (zakdma) [inline app/code/Magento/Sales/Test/Mftf/ActionGroup/AssertAdminPaymentMethodRadioButtonExistsOnCreateOrderPageActionGroup.xml]: Can we check particular payment radio by payment method name?

File: app/code/Magento/Sales/Test/Mftf/Test/AdminCheckingPaymentMethodRadioButtonPresentAfterReloadOrderPageTest.xml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            <argument name="customer" value="$createCustomer$"/>
+        </actionGroup>
+
+        <!-- Add Simple product to order -->
+        <actionGroup ref="AddSimpleProductToOrderActionGroup" stepKey="addSimpleProductToTheOrder">
+            <argument name="product" value="$createProduct$"/>
+        </actionGroup>
+
+        <!-- Assert label with radio button presents on the page -->
+        <actionGroup ref="AssertAdminPaymentMethodRadioButtonExistsOnCreateOrderPageActionGroup" stepKey="assertPaymentRadioButtonIsPresent"/>
```
Comment (zakdma) [inline app/code/Magento/Sales/Test/Mftf/Test/AdminCheckingPaymentMethodRadioButtonPresentAfterReloadOrderPageTest.xml]: Can we check both payment methods exists?

**Review by zakdma — CHANGES_REQUESTED** [review]

**Comment by victorpetryk** [comment]
@magento run all tests

**Review by zakdma — APPROVED** [review]

**Comment by engcom-Oscar** [comment]
:heavy_check_mark: QA passed

Was checked case from Manual testing scenarios 
:x: **Before:**
Screenshot from 2021-02-12 16-04-20

:heavy_check_mark: **After:**
Screenshot from 2021-02-12 16-18-23

**Comment by victorpetryk** [comment]
@magento run Functional Tests B2B

**Comment by victorpetryk** [comment]
@magento run all tests

**Comment by zakdma** [comment]
@magento import code to https://github.com/magento-tsg/magento2ce

**Comment by magento-engcom-team** [comment]
@zakdma, an error occurred during the Pull Request import.

**Comment by magento-engcom-team** [comment]
@zakdma, an error occurred during the Pull Request import.

**Comment by zakdma** [comment]
@magento import code to https://github.com/magento-tsg/magento2ce

**Comment by magento-engcom-team** [comment]
@zakdma, an error occurred during the Pull Request import.

**Comment by magento-engcom-team** [comment]
@zakdma, an error occurred during the Pull Request import.

**Comment by zakdma** [comment]
@magento import code to https://github.com/magento-tsg/magento2ce

**Comment by magento-engcom-team** [comment]
@zakdma, an error occurred during the Pull Request import.

**Comment by magento-engcom-team** [comment]
@zakdma, an error occurred during the Pull Request import.

**Comment by zakdma** [comment]
@magento import code to https://github.com/magento-tsg/magento2ce

**Comment by magento-engcom-team** [comment]
@zakdma the branch with code successfully imported into`magento-tsg/magento2ce` repository. Branch name: `imported-magento-magento2-32107`.

**Pull request merged**